yt_dlp/extractor/orf.py

```diff
@@ -565,3 +567,71 @@ def _real_extract(self, url):
+    _TESTS = [{
+        'url': 'https://on.orf.at/video/14210000/school-of-champions-48',
+        'info_dict': {
+            'id': '14210000',
+            'ext': 'mp4',
+            'duration': 2651.08,
+            'thumbnail': 'https://api-tvthek.orf.at/assets/segments/0167/98/thumb_16697671_segments_highlight_teaser.jpeg',
+            'title': 'School of Champions (4/8)',
+            'description': 'md5:d09ad279fc2e8502611e7648484b6afd',
+            'media_type': 'episode',
+        }
+    }]
+
+    def _call_api(self, video_id, display_id):
+        # NOTE: the prefix `3dSlfek03nsLKdj4Jsd` is only based on my observation on several
+        # api call. This string may change in future
+        encrypted_id = base64.b64encode(f'3dSlfek03nsLKdj4Jsd{video_id}'.encode()).decode()
+        api_json = self._download_json(
+            f'https://api-tvthek.orf.at/api/v4.3/public/episode/encrypted/{encrypted_id}', display_id)
+
+        formats, subtitles = [], {}
+        for manifest_type in api_json.get('sources') or [{}]:
+            for manifest_info in traverse_obj(api_json, ('sources', manifest_type, ...)):
+                fmt, subs = [], {}
+                if manifest_type == 'hls':
+                    fmt, subs = self._extract_m3u8_formats_and_subtitles(manifest_info.get('src'), display_id)
+                elif manifest_type == 'dash':
+                    fmt, subs = self._extract_mpd_formats_and_subtitles(manifest_info.get('src'), display_id, fatal=False)
+                else:
+                    continue
+                formats.extend(fmt)
+                self._merge_subtitles(subs, target=subtitles)
+
+        return {
+            'id': video_id or api_json.get('id'),
+            'formats': formats,
+            'subtitles': subtitles,
+            **traverse_obj(api_json, {
+                'duration': ('duration_second', float_or_none),
+                'title': (('title'), ('headline')),
+                'description': (('description'), ('teaser_text')),
+                'media_type': 'video_type'
+            })
+        }
+
+    def _real_extract(self, url):
+        video_id, display_id = self._match_valid_url(url).group('id', 'slug')
+        webpage = self._download_webpage(url, display_id)
+        json_ld_data = self._search_json_ld(webpage, display_id)
+
+        api_data = self._call_api(video_id, display_id)
+
+        return merge_dicts(api_data, {
+            'id': video_id,
+            'title': (json_ld_data.get('title')
+                      or self._html_search_meta(['og:title', 'twitter:title'], webpage)),
+            'description': (json_ld_data.get('description')
+                            or self._html_search_meta(['description', 'og:description', 'twitter:description'], webpage)),
+            **traverse_obj(json_ld_data, {
+                'duration': ('duration', {float_or_none}),
+                'timestamp': ('timestamp', int_or_none),
```

Comment:
```suggestion
                'timestamp': ('timestamp', {int_or_none}),
```
not really necessary to transform though since `_search_json_ld()` should return the proper types

Reply:
Resolved in https://github.com/yt-dlp/yt-dlp/pull/9113/commits/5f576e600b54f7d7c58fbbb4bb73e0d20370e6a7